dbclient/ClustersClient.py

```diff
@@ -230,10 +234,16 @@ def import_cluster_configs(self, log_file='clusters.log', acl_log_file='acl_clus
             return
         current_cluster_names = set([x.get('cluster_name', None) for x in self.get_cluster_list(False)])
         old_2_new_policy_ids = self.get_new_policy_id_dict()  # dict of {old_id : new_id}
+        error_logger = logging_utils.get_error_logger(
+            wmconstants.WM_IMPORT, wmconstants.CLUSTER_OBJECT, self.get_export_dir())
+        checkpoint_cluster_configs_set = self._checkpoint_service.get_checkpoint_key_set(
+            wmconstants.WM_IMPORT, wmconstants.CLUSTER_OBJECT)
         # get instance pool id mappings
         with open(cluster_log, 'r') as fp:
             for line in fp:
                 cluster_conf = json.loads(line)
+                if 'cluster_id' in cluster_conf and checkpoint_cluster_configs_set.contains(cluster_conf['cluster_id']):
+                    continue
```

Comment:
Log something when skipping. I think it'll help users understand the behavior. The same to other places.

Reply:
Make sesnse!

Reply:
Actually, it's already logged as part of the "contains" message, just like other checkpointer logics